plugins/module_utils/aci.py

```diff
@@ -1657,7 +1657,9 @@ def dump_json(self):
 
             elif self.params.get("state") == "absent" and self.existing:
                 for aci_class in self.existing[0]:
-                    mo[aci_class] = dict(attributes=dict(dn=dn_path, status="deleted"))
+                    attributes = deepcopy(self.existing[0][aci_class]["attributes"])
```

Comment:
Do we need to include all the attributes or will just name do?

Reply:
This change is still to be decided because it is actually a NDI issue (bug already identified). 

However if we do decide to make changes name will likely not be suffice. I think the identifier of the class is required by NDI, triggering the error of name for name attribute. But other classes can have different identifiers, thus I think more attributes should be exposed. Since we don't keep track of these identifier attributes we either expose all ( generic few lines of code ) or need to make changes per module to add the identifier attributes to the mo output.